docker-compose.ssl.yml

```diff
@@ -0,0 +1,32 @@
+version: "2.1"
+services:
+  webssl:
+    extends:
+      file: docker-compose.base.yml
+      service: webbase
+    command: /bin/bash -c "scripts/generate_dev_ssl_cert.sh /etc/nginx/ssl localhost && gunicorn -w 4 --bind 0.0.0.0:8000 --access-logfile=- --timeout=120 --worker-class=meinheld.gmeinheld.MeinheldWorker kuma.wsgi:application"
+    environment:
+      - ACCOUNT_DEFAULT_HTTP_PROTOCOL=https
+      - ALLOWED_HOSTS=localhost
+      - ATTACHMENT_HOST=localhost
+      - CSRF_COOKIE_SECURE=True
+      - PROTOCOL=https://
+      - SESSION_COOKIE_SECURE=True
+      - SITE_URL=https://localhost
+    ports:
+      - "8443:8000"
```

Comment:
Why `8443` port?

Reply:
443 is the SSL port, so 8443 makes some sense. There's no rule that I know of, just what I've used on other projects.  It's for debugging, and could be removed.

Reply:
I think this port should not be exposed. As its already running the `nginx` container that can connect the `gunicorn` with internal network.